.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,7 +1,7 @@
 contact_links:
   - name: Esri Support
-    url: https://support.esri.com/
-    about: Bugs in released versions of the SDK.
-  - name: ArcGIS Ideas
-    url: https://community.esri.com/t5/forums/searchpage/tab/message?advanced=false&allow_punctuation=false&q=javascript
-    about: Share your ideas with the global Esri community.    
\ No newline at end of file
+    url: https://support.esri.com/en/contact-tech-support
+    about: Report bugs in released versions of the SDK.
```

Comment:
Should we say something about requesting enhancements through Esri support here?

Reply:
Done!